crates/chia-bls/src/bls_cache.rs

```diff
@@ -122,9 +129,9 @@ impl BlsCache {
         let msgs = msgs
             .iter()?
             .map(|item| item?.extract())
-            .collect::<PyResult<Vec<PyBackedBytes>>>()?;
```

Comment:
what's the rationale for making this change?

Reply:
It felt better passing a pubkey in rather than serializing and deserializing, but I wasn't aware that that was just causing it to be pickled, same as with the GTElement change. 
I've reverted it back now.